js/webform_civicrm_membership_extras.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @file
+ * JavaScript .
+ */
+
+ (function ($) {
+  Drupal.behaviors.acu_webform_draft_group = {
```

Comment:
`acu_webform_draft_group` ? seems you tested your fix on certain client site which have such module.

Reply:
fixed now chief

Reply:
thanks, but I would like to see another screenshot tested on a standard compuclient site.